README.md

```diff
@@ -1,2 +1,71 @@
 # docker-itchysats
-ItchySats in a Docker container
+
+A repo facilitating running ItchySats inside a [Docker][https://www.docker.com/what-docker] container on a x86-64 (e.g. macOS, Linux, Windows) or ARMv7 system.
+Docker website provides extensive tutorials on Docker installation, please refer there if you are new to Docker.
+
+The following guide assumes that you have Docker running, having [docker-compose][https://docs.docker.com/compose/install/] installed is also recommended.
+
+# Quickstart
+
+## Testnet
+
+If you just want to check out the application, and want to try out a "demo" version, where you don't trade with real money, then this config is for you.
+
+```
+docker-compose project-directory=testnet up -d
+```
+
+## Mainnet
+
+If you're ready to trade with real money, use the following command:
+
+```
+docker-compose project-directory=testnet up -d
+```
+
+## User interface
+
+Docker command starts an ItchySats service running in the background.
+The service can be controlled via the provided web UI accessible on [https://localhost:8000][https://localhost:8000]
+Web UI requires authentication via password generated from your `taker_seed`.
```

Comment:
You can expect people don't know what you mean with `taker_seed`.

Reply:
good point. looking at the main repo, we lack a clear explanation there either - I'll add it to FAQ and link it there.

Reply:
https://github.com/itchysats/itchysats/pull/2210 (will link it here when FAQ gets merged)